src/snowflake/snowpark/dataframe.py

```diff
@@ -1918,6 +1919,9 @@ def agg(
             - :meth:`DataFrame.group_by`
         """
 
+        # Check exprs arguments.
```

Comment:
This comment doesn't add much value thanks to all the well chosen names in this context.